### PR TITLE
Added the SNAPSHOT version

### PR DESCRIPTION
**Pull request by ngocmanhha**
- Updated scm configuration
- Added developer info
- Added distributionManagement configuration
- Updated the jackson-databind version to fixup vulnerability issue

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/java-libraries/job/test-results-aggregator-plugin-tests-zuul-el7/5/ : SUCCESS in 31s
- https://checklist.intgdc.com/job/compliance/job/test-results-aggregator-plugin-oss-check/4/ : SUCCESS in 6s

File: pom.xml [hunk 1]
```diff
@@ -1,91 +1,114 @@
...
+        <developer>
+            <id>sdrss</id>
+            <name>Socrates Sidereas</name>
+            <email>socrates.sidereas@gmail.com</email>
+        </developer>
+		<developer>
+			<id>ngocmanhha</id>
+			<name>Manh Ha</name>
+			<email>manh.ha@gooddata.com</email>
+		</developer>
```
Comment (helobinvn) [inline pom.xml:57]: fix indention

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/test-results-aggregator-plugin-fossa-scanning-zuul/2/ : FAILURE in 2m 19s (non-voting)
- https://checklist.intgdc.com/job/java-libraries/job/test-results-aggregator-plugin-tests-zuul-el7/7/ : SUCCESS in 1m 47s
- https://checklist.intgdc.com/job/compliance/job/test-results-aggregator-plugin-merge-compliance-check/2/ : SUCCESS in 21s

**Pull request merged**